packages/perseus/src/widgets/radio/choice-icon.jsx

```diff
@@ -57,23 +57,28 @@ class SATChoiceIcon extends React.Component<{
             color = styleConstants.white;
             backgroundColor = styleConstants.satBlue;
         }
-        return {color, backgroundColor, borderColor};
+
+        let borderRadius;
+        if (multipleSelect) {
+            borderRadius = 3;
+        } else {
+            borderRadius = SAT_ICON_SIZE;
+        }
+
+        return {color, backgroundColor, borderColor, borderRadius};
     }
 
     render(): React.Node {
-        const {letter, a11yText, reviewMode, checked, correct, crossedOut} =
-            this.props;
-        const {color, backgroundColor, borderColor} = this.constructStyles(
-            reviewMode,
-            correct,
-            checked,
-        );
+        const {letter, a11yText, crossedOut} = this.props;
+        const {color, backgroundColor, borderColor, borderRadius} =
```

Comment:
**style nit:** instead of destructuring all of these values then restructuring them below, you could do:
```
const {color, ...style} = this.constructStyles();
```
and then do
```
style={style}
```
below.

Reply:
I tried this, but it looks like borderColor is also used elsewhere. I know I could do:

``` JS
const {color, borderColor, ...style} = this.constructStyles();
// ....
style={{...style, borderColor}}
```

But I think that's starting to get needlessly complex.

Reply:
That's too bad.